docs/TODO

```diff
@@ -125,7 +125,6 @@
  15. Schannel
  15.1 Extend support for client certificate authentication
  15.2 Extend support for the --ciphers option
- 15.3 Add option to disable client certificate auto-send
  15.4 Add option to allow abrupt server closure
```

Comment:
This should probably be 15.3 now? =)

Reply:
No we tend to leave them as is until removed rather than change the number, since they may be referenced elsewhere.